_static/files/command-name/command-name.rst

```diff
@@ -0,0 +1,27 @@
+**To list the available widgets**
+
+The following ``command-name`` example lists the available widgets in your AWS account. ::
+
+    aws awesomeservice command-name \
+        --parameter1 file://myfile.json \
+        --parameter2 value2 \
+        --lastparam lastvalue
+
+Contents of ``myfile.json``::
+
+    {
+        "somekey": "some value"
+    }
+
+Output::
+
+    {
+        "All of the output": "goes here",
+        "More Output": [
+            { "arrayitem1": 1 },
+            { "arrayitem2": 2 }
+        ]
+        "Each indent": "Must be 4 spaces"
+    }
+
+For more information, see `This is the topic title <https://link.to.the/topic/page>`__ in the *Name of your guide*.
```

Comment:
It looks like the link checker that we run on the github pages is flagging this as an invalid URL. Could we update this to a valid one? Otherwise, I can look to see whether we can ignore files in the `_static` directory.